campari/run_one_object.py

```diff
@@ -1,26 +1,27 @@
 # Standard Library
 import pathlib
+
```

Comment:
Why the blank line?  `pathlib` and `warnings` are both part of the standard library.  They should be one block.